save-backend/src/main/kotlin/org/cqfn/save/backend/scheduling/StandardSuitesUpdateScheduler.kt

```diff
@@ -39,11 +40,12 @@ class UpdateJob(
  * A component that is capable of scheduling [UpdateJob]
  */
 @Service
-@Profile("automatic-updates")
+@Profile("prod")
 class StandardSuitesUpdateScheduler(
     private val scheduler: Scheduler,
     configProperties: ConfigProperties,
 ) {
+    private val logger = LoggerFactory.getLogger(javaClass)
```

Comment:
`getLogger(StandardSuitesUpdateScheduler::class`)?

Reply:
That's essentially the same, I thought that using `this.javaClass` instead of explicit `this` class name can be more compact